data/src/entity_manager/system.py

```
@@ -3983,7 +3994,7 @@ def _verify_result(self, entity_class, id_value, cursor):
         # creates the final verified structure, in case the provided
         # id value is a tuple a tuple should be creates from the verifications
         # list otherwise the first result should be used (single value)
-        verified = id_value_type == tuple and tuple(verifications) or verifications[0]
+        verified = tuple(verifications) if id_value_type == tuple else verifications[0]
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)